src/TestGymMemberTracker.java

```diff
@@ -0,0 +1,101 @@
+// --== Test Gym Member Tracker ==--
+// Name: Christopher Nguyen
+// Email: cnguyen29@wisc.edu
+// Team: AA
+// TA: Sophie Stephenson
+// Lecturer: Florian Heimerl
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+import java.util.IllegalArgumentException;
+
+public class TestGymMemberTracker{
+	/**
+	 * Tests importation of the CSV file and
+	 * checks for correct members
+	 */	
+	@Test	
+	public void testImportCSV(){
+	  State expected = null;
+	  expectedMembers(expected);
+	  State returned = new State();
+	  if (!returned.toString().equals(expected.toString()){
```

Comment:
replace with assertEquals(expected.toString(), returned.toString(), "Members in file do not match expected members");